### ISSUE TITLE
Adaptive Live playlist

### ISSUE THREAD

**Issue by ratiboo** [kaltura/nginx-vod-module#1282]
I have only managed to make the adaptive playlist work to live, with the first file
I have tried creating a json playlist that includes 3 qualities, per file, file_720.mp4, file_480.mp4 and file_240.mp4 and for the second file file2_720.mp4, file2_480.mp4 and file2_240.mp4, it doesn't work.
Does the software support adaptive live playlist? or is it not possible?

**Comment by erankor** [comment]
yes, it's supported, there's a sample php script that shows how to create the json (https://github.com/kaltura/nginx-vod-module/blob/master/test/playlist.php) to add more qualities, just need to add more files to the array, e.g. -
```
$filePaths = array(
	// clip 1
	array(
		'/path/to/video1-low.mp4',
		'/path/to/video1-high.mp4',
		'/path/to/subtitles1.srt',
	),
	// clip 2
	array(
		'/path/to/video2-low.mp4',
		'/path/to/video2-high.mp4',
		'/path/to/subtitles2.srt',
	),
);

$languages = array(
	null,
	null,
	'eng',
);
```
The number of renditions must be aligned across all clips.

**Comment by ratiboo** [comment]
thanks!!!

Do you have a example json??

I cant do playlist.php work :/

Enviado desde mi iPhone
